lib/run_loop/simctl.rb

```diff
@@ -207,6 +207,49 @@ def wait_for_shutdown(device, timeout, delay)
       in_state
     end
 
+    def boot(device)
+      if simulator_state_as_int(device) == SIM_STATES["Booted"]
+        RunLoop.log_debug("Simulator is already shutdown")
```

Comment:
s/is already shutdown/is already booted/ ?

Reply:
Done